protocols/relay/src/behaviour/handler.rs

```diff
@@ -931,3 +979,130 @@ pub(crate) struct CircuitParts {
     dst_stream: Stream,
     dst_pending_data: Bytes,
 }
+
+struct StopCommand {
+    circuit_id: CircuitId,
+    inbound_circuit_req: CircuitReq,
+    src_peer_id: PeerId,
+    src_connection_id: ConnectionId,
+    max_circuit_duration: Duration,
+    max_circuit_bytes: u64,
+}
+
+pub struct ReservationReq {
```

Comment:
Why move this into here? I'd prefer to keep this file shorter to make it easier to navigate.

Reply:
> I'd prefer to keep this file shorter to make it easier to navigate.

Ok, got it.

> Why move this into here?

Because functions which use them is here ))
If we move these `fn`s to `inbound_hop` then structs would move too.

Reply:
> > Why move this into here?
> 
> Because functions which use them is here ))

Following this logic, everything should be in one file because everything is eventually used by `fn main` :D

Previously, they were in a sub-module of `handler` which I find to be a meaningful structure, i.e. only `handler` needs these structs / functions and only `handler` has access to its own submodules.

Reply:
> Following this logic, everything should be in one file because everything is eventually used by fn main :D

Compiler gathers everything in one file, eventually )))

Reply:
I will fix your comments step-by-step. Thanks!